src/patternfly/components/DataList/data-list-item-controls.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-data-list__item-controls{{#if data-list-item-controls--modifier}} {{data-list-item-controls--modifier}}{{/if}}"
```

Comment:
let's rename this to `pf-c-data-list__item-control` and fix the `--attribute` names below.